Presentation.Web/Controllers/API/V1/OData/DataProcessingRegistrationReadModelsController.cs

```diff
@@ -21,7 +25,7 @@ public DataProcessingRegistrationReadModelsController(IDataProcessingRegistratio
         }
 
         [EnableQuery]
-        [RequireTopOnOdataThroughKitosToken]
+        [SwaggerResponse(HttpStatusCode.OK, type:typeof(ODataListResponse<DataProcessingRegistrationReadModel>))]
```

Comment:
using the ODataListResponse attribute ensures we get the right response object with the odd `@odata.*` properties along with the `value` property containing the model definition